### DISCUSSION TITLE
Which skills are needed to the frontend developer to work in web3/crypto/blockchain company?

### DISCUSSION THREAD

**Discussion by vladimirskayatamara** [community/community#53526] [Programming Help] [answered]
### Body

Which skills are needed to the frontend developer to work in web3/crypto/blockchain company?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by its-a-setup** [comment, marked as answer]
I would say that on top of standard frontend skills, you'll need the following:
1. Understanding of web3 technologies and concepts such as Ethereum, Solidity, IPFS, smart contracts, dApps, and DEXs.
2. Experience with Web3.js, Ethers.js, Remix, MetaMask, and Ganache.

And of course passion for web3!
